kubernetes/kserve/kserve_wrapper/README.md

```diff
@@ -148,7 +148,7 @@ Output:
 The curl request for explain is as below:
 
 ```bash
-curl -H "Content-Type: application/json" --data @serve/kubernetes/kserve/kf_request_json/v1/mnist.json http://127.0.0.1:8080/v1/models/mnist:explain
+curl -H "Content-Type: application/json" --data @serve/kubernetes/kserve/kf_request_json/v1/mnist.json http://0.0.0.0:8080/v1/models/mnist:explain
```

Comment:
Please update steps to match the KServe docs. User needs to get correct values for SERVICE_HOSTNAME, INGRESS_HOST and INGRESS_PORT to run the actual predictions and explain requests:

https://kserve.github.io/website/0.11/modelserving/v1beta1/torchserve/#model-inference_1